Libraries/LibGfx/Painter.cpp

```diff
@@ -1300,6 +1300,8 @@ void Painter::fill_path(Path& path, Color color, WindingRule winding_rule)
     for (auto& segment : segments)
         draw_line(segment.from, segment.to, Color::from_hsv(++i / segments.size() * 255, 255, 255), 1);
 #endif
+
+    return;
```

Comment:
This seems useless

Reply:
lol there was a stroke there, I forgot to remove the return along with it :)